scripts/ci/run_integration_tests.sh

```diff
@@ -29,6 +29,10 @@ for target in ${TEST_TARGETS[@]:-}; do
   ((s=s+1))
 done
 
+# Stash system path in a special location that is picked up by test_runner.js
+# for inner bazel-in-bazel execution 
+echo "${PATH}" > /tmp/build_bazel_rules_nodejs__bazel_integration_test__orig_path
```

Comment:
I guess it's too annoying to encode the whole value into a `--test_arg`?

Reply:
Actually. This may be incorrect as its not working on linux anyway. Still experimenting.

Reply:
Yes. It is not needed. Misdiagnosed that one.